Gigs/Gigs/Application/GigController.swift

```diff
@@ -147,17 +147,17 @@ class GigController {
             
             do {
                 let gigNames = try decoder.decode([Gig].self, from: data)
-                completion(.success(gigNames))
-                self.gigs.append(contentsOf: gigNames)
+                completion(.success(true))
+                self.gigs = gigNames
```

Comment:
If the `gigs` array already has the gigs you fetched previously, appending the newly fetched gigs would double a lot of the gigs.